src/pages/Lend/Filter/FilterMenuSection.vue

```diff
@@ -1,19 +1,25 @@
 <template>
-	<div class="filter-menu-section" :class="{open}">
-		<div class="filter-summary" @click="toggleMenu">
-			<div class="filter-summary-title" >
-				{{ title }}
+	<div class="filter-menu-section" :class="{open}" @click="toggleMenu">
+		<div class="filter-summary">
+			<div class="filter-summary-title-row">
+				<div class="filter-summary-title" >
+					{{ title }}
+				</div>
+				<kv-icon class="filter-summary-title-chevron" name="small-chevron-mobile" />
 			</div>
 			<div class="filter-summary-applied-filters">{{ appliedFilters }}</div>
 		</div>
-		<div class="filter-items-container">
+		<div class="filter-items-container" @click="handleClickFilterItemsContainer">
```

Comment:
Basically I increased the click area for toggling the filter to include the padding. This handler adds a `stopPropagation` so clicking on filters themselves doesn't toggle the menu.

Reply:
This is fine. Just wondering if @click.stop="" would have worked too?

Reply:
Nice! It does and we can delete 3 lines of code :+1: